src/crimson/osd/osd.cc

```diff
@@ -919,10 +922,21 @@ void OSD::handle_conf_change(
   const crimson::common::ConfigProxy& conf,
   const std::set <std::string> &changed)
 {
-  if (changed.count("osd_beacon_report_interval")) {
+  if (changed.contains("osd_beacon_report_interval")) {
     beacon_timer.rearm_periodic(
       std::chrono::seconds(conf->osd_beacon_report_interval));
   }
+  if (changed.contains("osd_op_complaint_time")) {
+    local_conf().set_val(
+      "osd_op_complaint_time", std::to_string(conf->osd_op_complaint_time)
+    ).get();
+  }
+  if (changed.contains("osd_aggregated_slow_ops_logging")) {
+    local_conf().set_val(
+      "osd_aggregated_slow_ops_logging",
+      conf->osd_aggregated_slow_ops_logging ? "true" : "false"
+    ).get();
+  }
```

Comment:
We query the conf each time we `get_health_metrics`, `osd_op_complaint_time` and `osd_aggregated_slow_ops_logging` shouldn't be added tp `handle_conf_change`. Sorry for the mistake in the previous comments.